cmd/jag/commands/setup.go

```diff
@@ -54,6 +55,31 @@ func SetupCmd(info Info) *cobra.Command {
 		SilenceUsage: true,
 		RunE: func(cmd *cobra.Command, args []string) error {
 			ctx := cmd.Context()
+			check, err := cmd.Flags().GetBool("check")
+			if err != nil {
+				return err
+			}
+
+			if check {
```

Comment:
Will returning with an error from there give an exit code of 1?

Reply:
yes.

Reply:
not it will :)